tests/testthat/test-object-package.R

```diff
@@ -17,3 +17,94 @@ test_that("person turned into meaningful text", {
     person_desc(comment = c("ORCID" = "1234", "extra"))
   })
 })
+
+test_that("package description not affected if no links", {
+  text <- "A simple description with no links."
+
+  parsed <- package_description_urls(text)
+
+  expect_equal(
+    parsed,
+    text
+  )
+  expect_snapshot(parsed)
+})
+
+test_that("can autolink urls on package Description", {
+  urls <- paste(
+    "<https://github.com/>",
+    "Secured <https://github.com/>.",
+    "No link <www.github.com/>.",
```

Comment:
I'd suggest moving the all examples that shouldn't be linked into one test case.